pom.xml

```diff
@@ -163,25 +189,49 @@
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>mariadb</artifactId>
-      <version>1.10.6</version>
+      <version>${test-containers}</version>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>mysql</artifactId>
-      <version>1.10.6</version>
+      <version>${test-containers}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>oracle-xe</artifactId>
+      <version>${test-containers}</version>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>postgresql</artifactId>
-      <version>1.10.6</version>
+      <version>${test-containers}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <version>2.0.1</version>
```

Comment:
Last release 2.1.1 ?

Reply:
2.1.1 is used in WaarpCommon and it gives me compilation warnings because the jars of rs-api distributed on Maven central are built for java 8+ (`major version 52 is newer than 51, the highest major version supported by this compiler.`).
So we are stuck with 2.0.x to support Java 6.

*Sidenote*: for the releases and tests, I always use the lowest version of java supported by maven (which is Java 7) because there are incompatibilities in the stdlib of different versions of Oracle JVM which generates runtime errors.
More precisely, the classes are compiled into bytecode for the target version of java, which is 6, using the method signatures from the stdlib of the JDK used to build it. There are differences in methods signatures between Java 6 and Java 8. Compiling R66 classes with Java 8 results in runtime method not found exceptions, because the methods referenced in the bytecode are not found in java 6 stdlib.

On the same topic :
* The version of h2 used by R66 is 1.4.197
* The version of h2 used by Common is 1.4.197

And I guess there are more like that.

**Can we all agree to leave dependencies in this PR and to address this issue (dependency upgrades, using the same versions of the external dependencies in all the Waarp* jars, etc) in another PR (actually 1 PR per Waarp* project) ?**

Reply:
OK for me ;-)